### DISCUSSION TITLE
Not able to push

### DISCUSSION THREAD

**Discussion by Osomhe1** [community/community#53583] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

trying to push to a branch but it doesn't go through and does not return an error no git command is working except git add and commit, the rest does not work. How can I resolve it please?

**Comment by airtower-luna** [comment]
Please include the *full* commands you use and their output, otherwise it's impossible to say what may be going on.

**Reply by Osomhe1** [reply]
![Screenshot (832)](https://user-images.githubusercontent.com/64287450/234140574-d1343830-937f-4927-ae76-104711a7e14f.png)

**Reply by airtower-luna** [reply]
Did you cancel the `git push` after a while? If yes, after how long? It definitely shouldn't terminate with no output. :sweat_smile: 

If it just hangs there might be some network issue, which would be hard to figure out remotely. You could check `git remote -v` to make sure the repository URL is correct.

**Reply by Osomhe1** [reply]
No, I did not cancel it; that is how it behaves

**Reply by airtower-luna** [reply]
That's really odd, it should either actually do the push or show an error message. The only things that I can still think of is checking the remote URL (as mentioned above) *and* the Git version (`git --version`) to make sure you're not running an outdated verion. 2.40.1 is the latest Git version right now.

**Comment by Osomhe1** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> trying to push to a branch but it doesn't go through and does not return an error no git command is working except git add and commit, the rest does not work. How can I resolve it please?

![Screenshot (832)](https://user-images.githubusercontent.com/64287450/234140497-6d2376e5-4c50-4588-961e-0ea29d24e6af.png)
